<launch>
    <node pkg="rosbag" type="play" name="bag_player" 
          args="--clock $(find fsd_common_msgs)/bag/cone_detected.bag" 
          output="screen" required="true"/>
    <node pkg="fsd_common_msgs" type="cone_projection_node" name="cone_projection_node" 
          output="screen" required="true"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find fsd_common_msgs)/rviz/fsd_visualization.rviz" />
</launch>
